mobile/src/libs/components/card/card.tsx

```diff
@@ -9,49 +10,58 @@ import {
   Text,
   View,
 } from '#libs/components/components';
+import { AppColor } from '#libs/enums/enums';
 import { type IconName } from '#libs/types/types';
 
 import { DEFAULT_NUMBER_OF_LINES } from './libs/constants/constants';
 import { styles } from './styles';
 
 type Properties = {
   title: string;
-  id?: string;
   image?: ImageSourcePropType;
   iconName?: IconName;
   iconColor?: string;
-  onPress: (title: string, id?: string) => void;
+  onPress: () => void;
+  iconRight?: IconName;
+  onIconPress?: () => void;
 };
 
 const Card: React.FC<Properties> = ({
   title,
-  id = '',
   image = imagePlaceholder,
   iconName,
   iconColor,
   onPress,
+  iconRight,
+  onIconPress,
 }) => {
-  const handlePress = (): void => {
-    onPress(title, id);
+  const renderRightSwipeActions = (): React.ReactNode => {
+    return iconRight ? (
```

Comment:
```suggestion
    return Boolean(iconRight) && (
```

Reply:
@v0ldemar01 if i do so i will have an error for icon name, because it can be undefined and i should render condition only  when it is truthy
![Снимок экрана 2023-09-28 142701](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/111873795/85444d90-202d-483a-8f67-aea63a2db502)

Reply:
Use type assertion